2-structured-data/books/model-datastore.js

```diff
@@ -95,9 +95,9 @@ module.exports = function(config) {
       .order('title')
       .start(token);
 
-    ds.runQuery(q, function(err, entities, cursor) {
+    ds.runQuery(q, function(err, entities, nextQuery) {
       if (err) { return cb(err); }
-      var hasMore = entities.length === limit ? cursor : false;
+      var hasMore = entities.length === limit ? nextQuery.startVal : false;
```

Comment:
This is a little awkward, since it's not a documented property. However, I think this is an okay way to access it, similar to `file.name` on a File object. Do you have any ideas how we could make this better?